app/views/reviews.py

```diff
@@ -20,25 +19,34 @@ class BusinessReviews(Resource):
     @jwt_required
     def post(self, business_id):
         """Add a business review.
-
-        Args:
-            business_id (int): business id parameter should be unique to identify each business.
-
-        Returns:
-            A success message to indicate review was added successfully.
-
-        Raises:
-            KeyError when the business id does not exist.
-            Error message when business review is empty.
-            Error message when no business id was provided.
-
+        ---
+        tags:
+            -   business reviews
+        parameters:
+            -   in: body
+                name: body
+                schema:
+                    $ref: '#/definitions/User'
+        security:
+            $ref: '#/components/securitySchemes/BearAuth'
+        responses:
+            201:
+                description: Created
+                content:
+                    application/json:
+                        schema:
+                            type: object
+                            properties:
+                                response_message:
+                                    type: string
+                                    description: message to show review has been added successfully
         """
         request_data = request.get_json(force=True)
         business_review = request_data['review']
```

Comment:
Using dictionary key-value pair for data extraction can lead to errors should on of the keys be missing.
It's advisable to use `request.get_json().get('variable')`  e.g. `request_data.get('review')`